mongod.conf

```diff
@@ -0,0 +1,11 @@
+storage:
+  dbPath: /data/db
+
+systemLog:
+  destination: file
+  path: /data/configdb/log/mongod.log
+  logAppend: true
+  logRotate: reopen
+
+replication:
```

Comment:
Remove this one currently, for prover node currently we may not use the replica set currently.